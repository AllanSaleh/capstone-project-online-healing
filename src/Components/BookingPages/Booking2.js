import React from 'react';
import QuestionComponent from './QuestionComponent';

const questionData = {
  question: 'What is your relationship status?',
  choices: [
    {
      id: 1,
      content: 'Single',
      selected: true,
    },
    {
      id: 2,
      content: 'Married',
      selected: false,
    },
    {
      id: 3,
      content: 'Divorced',
      selected: false,
    },
  ],
};

const Booking2 = () => (
  <div className="px-sides pt-navbar">
    <h1 className="text-3xl lg:text-title uppercase mb-4"> letâ€™s match you with the right therapist</h1>
    <h3 className="text-md lg:text-subtitle opacity-50 text-justify">
      Please fill out this short questionnaire to provide some general and anonymous background
      about you and the issues you'd like to deal with in online therapy. It would help us match you
      with the most suitable therapist for you.
    </h3>

    <div className="flex flex-col justify-between w-full md:max-w-md lg:max-w-2xl my-16 mx-auto p-8 shadow-md">
      <QuestionComponent question={questionData.question} choices={questionData.choices} />

      <div className="flex justify-between mt-16">
        <button
          type="button"
          className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition duration-300"
        >
          back
        </button>
        <button
          type="button"
          className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300"
        >
          next
        </button>
      </div>
    </div>
  </div>
);

export default Booking2;
