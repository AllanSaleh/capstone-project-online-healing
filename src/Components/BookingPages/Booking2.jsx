import React from 'react';
import { useHistory } from 'react-router-dom';

import QuestionComponent from './QuestionComponent';

const questionData = {
  question: 'What is your relationship status?',
  choices: [
    {
      content: 'Single',
    },
    {
      content: 'Married',
    },
    {
      content: 'Divorced',
    },
  ],
};

export default function Booking2() {
  window.scrollTo(0, 0);
  const history = useHistory();

  const PrevPage = () => {
    // Put firestore code here!
    history.push('/Booking1');
  };
  const NextPage = () => {
    // Put firestore code here!
    history.push('/Booking3');
  };

  return (
    <div className="h-firstsection px-sides pt-navbar">
      <h1 className="text-3xl lg:text-title uppercase my-8">
        {' '}
        letâ€™s match you with the right therapist
      </h1>
      <h3 className="text-md lg:text-subtitle opacity-50 text-justify">
        Please fill out this short questionnaire to provide some general and anonymous background
        about you and the issues you'd like to deal with in online therapy. It would help us match
        you with the most suitable therapist for you.
      </h3>

      <div className="flex flex-col justify-between w-full md:max-w-md lg:max-w-2xl my-16 mx-auto p-8 shadow-md">
        <QuestionComponent question={questionData.question} choices={questionData.choices} />

        <div className="flex justify-between mt-16">
          <button
            onClick={() => PrevPage()}
            type="button"
            className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition duration-300"
          >
            back
          </button>
          <button
            onClick={() => NextPage()}
            type="button"
            className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300"
          >
            next
          </button>
        </div>
      </div>
    </div>
  );
}
