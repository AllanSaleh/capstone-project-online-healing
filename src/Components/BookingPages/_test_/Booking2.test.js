import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Booking2 from '../Booking2';

it('Booking2 renders correctly', () => {
  const tree = renderer.create(<Booking2 />).toJSON();
  expect(tree).toMatchSnapshot();
});
