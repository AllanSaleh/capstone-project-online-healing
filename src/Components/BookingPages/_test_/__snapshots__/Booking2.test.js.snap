// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking2 renders correctly 1`] = `
<div
  className="h-firstsection px-sides pt-navbar"
>
  <h1
    className="text-3xl lg:text-title uppercase my-8"
  >
     
    letâ€™s match you with the right therapist
  </h1>
  <h3
    className="text-md lg:text-subtitle opacity-50 text-justify"
  >
    Please fill out this short questionnaire to provide some general and anonymous background about you and the issues you'd like to deal with in online therapy. It would help us match you with the most suitable therapist for you.
  </h3>
  <div
    className="flex flex-col justify-between w-full md:max-w-md lg:max-w-2xl my-16 mx-auto p-8 shadow-md"
  >
    <ul>
      <h1
        className="text-lg lg:text-subtitle"
      >
        What is your relationship status?
      </h1>
      <li
        className="my-4 p-4 lg:p-6 text-md lg:text-paragraph cursor-pointer bg-blue-dark text-white rounded-lg "
        value={1}
      >
        Single
      </li>
      <li
        className="my-4 p-4 lg:p-6 text-md lg:text-paragraph cursor-pointer bg-white border border-black border-opacity-20 rounded-lg "
        value={2}
      >
        Married
      </li>
      <li
        className="my-4 p-4 lg:p-6 text-md lg:text-paragraph cursor-pointer bg-white border border-black border-opacity-20 rounded-lg "
        value={3}
      >
        Divorced
      </li>
    </ul>
    <div
      className="flex justify-between mt-16"
    >
      <button
        className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition duration-300"
        onClick={[Function]}
        type="button"
      >
        back
      </button>
      <button
        className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300"
        onClick={[Function]}
        type="button"
      >
        next
      </button>
    </div>
  </div>
</div>
`;
